### PR TITLE
Add doc comments for public APIs (Part 1)

### PR DESCRIPTION
**Pull request by kateinoigakukun**

### CODE REVIEW

File: Sources/JavaScriptKit/FundamentalObjects/JSFunction.swift [hunk 1]
```diff
@@ -62,33 +86,59 @@ public class JSFunction: JSObject {
     }
 }
 
+/// `JSClosure` represents a JavaScript function whose body is written in Swift.
```
Comment (MaxDesiatov) [inline Sources/JavaScriptKit/FundamentalObjects/JSFunction.swift:89]:
```suggestion
/// `JSClosure` represents a JavaScript function the body of which is written in Swift.
```

File: Sources/JavaScriptKit/FundamentalObjects/JSObject.swift [hunk 2]
```diff
@@ -15,30 +39,49 @@ public class JSObject: Equatable {
...
 
     static let _JS_Predef_Value_Global: JavaScriptObjectRef = 0
+
+    /// A `JSObject` of the global scope object.
+    /// This allows access to the global properties and global names by accessing the `JSObject` returned.
     public static let global = JSObject(id: _JS_Predef_Value_Global)
 
     deinit { _release(id) }
 
+    /// Returns a Boolean value indicating whether two values points same objects.
```
Comment (MaxDesiatov) [inline Sources/JavaScriptKit/FundamentalObjects/JSObject.swift:80]:
```suggestion
    /// Returns a Boolean value indicating whether two values point to same objects.
```

**Review by MaxDesiatov — APPROVED** [review]
This is amazing, thank you so much 🙏 Just a couple of suggestions

**Comment by kateinoigakukun** [comment]
Thanks for wording suggestions 😄

**Pull request merged**